exercises/practice/strain/strain-test.ys

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env ys-0
+
+require ys::taptest: :all
+
+use: strain
+
+test::
+- name: Keep on empty list returns empty list
+  code: 'keep([] "fn([x] true)")'
```

Comment:
does it need to be a string? Not sure what the test mandates

Reply:
The single quotes not needed. e06fb78